src/main.ts

```diff
@@ -160,18 +160,26 @@ export class Main {
       setTimeout(function () {
         splash.destroy();
       }, 3000);
+    // splash.webContents.openDevTools();
     this.backgroundWindow = new BrowserWindow({
-      show: false,
+      show: true,
```

Comment:
is there a way to use feature-flag here? as we merge to main, we don't want this screen to appear until we have the complete flow functional.

Reply:
If we do show false, this screen will be hidden.